manage.py

```diff
@@ -17,6 +18,12 @@ def make_shell_context():
     "shell", Shell(
         make_context=make_shell_context))
 
+@app.errorhandler(404)
+def route_not_found(e):
+
+    return make_response(jsonify({
+        "error": 'The requested URL was not found on the server please check your spelling and try again'
```

Comment:
This error message can rather be shorter but the presence of this function is really GREAT, Thank you for this

Reply:
Thanks for the feedback.  I am going to make it shorter and descriptive.